app.test.js

```diff
@@ -46,4 +46,45 @@ describe('getCategories', () => {
             })
           });
       });
-})
\ No newline at end of file
+})
+
+describe('getReviews', () => {
+  test("Responds with an object containing an array of review objects", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+          expect(body.reviews).toEqual(expect.any(Array));
+        });
+    });
+    test("Responds with review objects with correct properties", () => {
```

Comment:
After deleting the first test, slightly tweak this description to mention you're testing it is an array.